README.md

```diff
@@ -1 +1,154 @@
-# expense_tracker_RoR
\ No newline at end of file
+# Expense_tracker_RoR
+
+# 📗 Table of Contents
+
+- [expense_tracker_RoR](#expense_tracker_RoR)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Expense_tracker_RoR ](#-expense_tracker_RoR-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Deployment](#deployment)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+    - [📝 License](#-license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Blog_app <a name="about-project"></a>
+
+**Expense_tracker_RoR** Expense_tracker_RoR app is a website that allows users to manage personal finances. Featuring secure awuthentication with Devise. It enables users to create their own expenses for different categories and all financial data remains private and secure.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Programming languages</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+ </details>
+
+<details>
+  <summary>Technologies</summary>
+  <ul>
+    <li><a href="https://git-scm.com/">Git</a></li>
+    <li><a href="https://github.com/">Github</a></li>
+    <li><a href="https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow">Gitflow</a></li>
+  </ul>
+ </details>
+ <details>
+  <summary>Frameworks</summary>
+  <ul>
+    <li><a href="https://getbootstrap.com/">Bootstrap 5</a></li>
+    </ul>
+ </details>
+<details>
+  <summary>Modules</summary>
+ </details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href='https://heroku.com/'>Heroku</a></li>
+    </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Add User, Categoty and Expense
+- List all Categories and Expenses
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/haliljon/expense_tracker_RoR.git
+
+  cd expense_tracker_RoR
+
+  bundle install
+
+  rails c
+
+  rspec spec --format documentation
+```
+
+### 🚀 Live Demo
+
+- [Demo Video](https://drive.google.com/file/d/1Kh__UMtrAGZR2UUCQ5z_Hbw7GDtPMo3Q/view?usp=sharing)
+
```

Comment:
Your presentation is a bit long and it is over the maximum time allowed

- [ ] Kindly record a new video and keep it under the maximum length of 5 mins

Reply:
Hi @AshakaE, I'll do that

Reply:
Hi @AshakaE from the last night till now I tried to deploy the app with heroku.com and render.com, both of them failed to deploy. Here are the screenshots, can you plz advice?

<img width="890" alt="Screenshot 2023-05-07 at 1 23 28 AM" src="https://user-images.githubusercontent.com/110017001/236645559-929618cf-547f-47a3-8f39-4d1032753b01.png">

<img width="1764" alt="Screenshot 2023-05-07 at 1 31 20 AM" src="https://user-images.githubusercontent.com/110017001/236645562-3e77d056-44c1-4549-bb2e-1663cf80ca2c.png">